src/components/pages/login/index.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import LoginForm from './loginForm';
-import logo from '../../../../images/landingPage/logo.png';
+import newBquest from '../../../../images/loginPage/white-bquest.svg.inline'
+import newBquestLogo from '../../../../images/loginPage/white-bquest-logo.svg.inline'
 
 class Login extends React.Component {
 
     render() {
         return(
             <div className="container-login">
                 <div className="logo">
-                    <img src={logo} id="logo" alt="logo" />
+
+					{/*<img className="logo__logo" src={newBquestLogo} alt="newBquestLogo"/>*/}
+					{/*<img className="logo__text" src={newBquest} alt="newBquest"/>*/}
+					<div className="logo__logo" dangerouslySetInnerHTML={{__html: newBquestLogo}} />
+					<div className="logo__text" dangerouslySetInnerHTML={{__html: newBquest}} />
```

Comment:
please fix indenting here. I thought that the .editorconfig file in the root of our repo would configure your/all text editors to obey this indenting but i guess not. I'll try to figure out why

Reply:
Seems Webstorm is not recognizing its file type... I'll see if I can fix it...

Reply:
Actually, I think the indenting here should be correct: I checked the file itself and seems indenting correctly, it is just at Github's comparison that makes it look longer